### PR TITLE
LF-2750: map routing bug

### PR DESCRIPTION
**Pull request by fsayyid**
Ticket: [LF-2750](https://lite-farm.atlassian.net/browse/LF-2750)

**TO TEST:** create at least one location from each of the following categories. The location should route to the appropriate tab. 
- **Crop enabled area:** Field, Green House, Garden
  - routes to crop tab
- **Non crop enabled area:** Barn, Ceremonial Area, Farm Site Boundary, Surface Water, Natural Area, Residence
  - routes to details tab
- **Crop enabled line:** Buffer Zone 
  - routes to crop tab
- **Non crop enabled line:** Fence, Water Course 
  - routes to details tab
- **Non sensor points (clustered and non-clustered):** Gate, WaterValve 
  - routes to details tab
- **Sensor (clustered and non-clustered):** Sensor 
  - routes to readings tab

### CODE REVIEW

**Review by Cali0707 — APPROVED** [review]
This looks good to me, and also seems to resolve the bug reported on https://lite-farm.atlassian.net/browse/LF-2632, which makes sense given that it looks like the issue was to do with clicking on gate locations.

File: packages/webapp/src/containers/Map/useSelectionHandler.js [hunk 1]
```diff
@@ -68,10 +68,14 @@ const useSelectionHandler = () => {
             });
             dispatch(canShowSelection(true));
             dispatch(locations(locationArray));
-          } else {
+          } else if (overlappedLocations.point[0].type === 'sensor') {
```
Comment (seinnlee) [inline packages/webapp/src/containers/Map/useSelectionHandler.js:71]: I should have used a constant on line 62. Can we have line 62 and 71 use constants for 'sensor' in this PR?

**Review by seinnlee — CHANGES_REQUESTED** [review]
Just one minor change requested. Otherwise, it looks good to me.

**Review by seinnlee — APPROVED** [review]
lgtm!

**Pull request merged**